api/openapi-spec/swagger.json

```diff
@@ -4634,6 +4634,10 @@
      "serial": {
       "description": "Serial provides the ability to specify a serial number for the disk device.\n+optional",
       "type": "string"
+     },
+     "tag": {
```

Comment:
I also checked this in the openstack documentation and it is not clear to me if this should allow an array of tags, since in the cloud-init workload we have `tags`.

Reply:
Yes, I think originally, the idea was to allow multiple tags per device, but I don't see the point of it. However, in the metadata `tags` is a list. I think we should present it as a list in metadata, but I'm not sure if we should allow setting multiple tags per device... what do you think?